packages/zcli-apps/src/commands/apps/new.ts

```diff
@@ -65,9 +65,9 @@ export default class New extends Command {
           if (err) {
             const fsExtraError = err as FsExtraError
             if (fsExtraError.code === 'ENOENT') {
-              reject(new Error(`Scaffold ${flagScaffold} does not exist: ${err}`))
+              return reject(new Error(`Scaffold ${flagScaffold} does not exist: ${err}`))
             }
-            reject(err)
+            return reject(err)
```

Comment:
Since reject will already exit the Promise executor function, the `return` is redundant

Reply:
Both option `{ overwrite: true, errorOnExist: true }` contradict each other? 
`errorOnExist` - it does fails if the directory exists
`overwrite` - override if the directory already exists
it it intended behaviour?

Reply:
@veera `errorOnExist` is only meaningful if overwrite is false. If overwrite is true, it will be overwritten regardless, making errorOnExist redundant. I've removed it now 👍